### DISCUSSION TITLE
PCM metrics documentation

### DISCUSSION THREAD

**Discussion by bellalzohir** [opcm/pcm#858] [Q&A]
Hello,

Could you please let me know where I can find the PCM metrics documentation?

Thank you!

**Comment by rdementi** [comment]
here are a few links:
https://github.com/intel/pcm/blob/master/doc/PCM-SENSOR-SERVER-README.md
https://www.intel.com/content/www/us/en/developer/articles/technical/intel-pcm-column-names-decoder-ring.html
https://raw.githubusercontent.com/wiki/opcm/pcm/Going-beyond-CPU-utilization-with-PCM-16-7-2020.pdf
